Source/Scene/Model.js

```diff
@@ -1267,6 +1292,79 @@ define([
 
     ///////////////////////////////////////////////////////////////////////////
 
+    function alignOffset(offset, alignment) {
+        offset = offset | 0;
+        alignment = alignment | 0;
+        return (Math.ceil(offset / alignment) * alignment) | 0;
+    }
+
+    function createDecompressedBuffers(model, context) {
+        var loadResources = model._loadResources;
+
+        if (loadResources.pendingBufferLoads !== 0) {
+            return;
+        }
+
+        var extensions = model.gltf.extensions;
+        if (!defined(extensions) || !defined(extensions.mesh_compression_open3dgc)) {
+            return;
+        }
+
+        var decompressedBuffers = model.gltf.extensions.mesh_compression_open3dgc.decompressedBuffers;
+        while (loadResources.decompressedBuffersToCreate.length > 0) {
```

Comment:
It's not our long-term throttling solution, but this should probably check `model.asynchronous` like [this](https://github.com/kainino0x/cesium/blob/mesh_compression_open3dgc/Source/Scene/Model.js#L1468) **unless** decompression is very fast.